libs/payments/customer/src/lib/util/determinePaymentMethodType.ts

```diff
@@ -22,20 +22,24 @@ export const determinePaymentMethodType = (
   customer?: StripeCustomer,
   subscriptions?: StripeSubscription[]
 ): PaymentMethodTypeResponse => {
-  if (customer?.invoice_settings.default_payment_method) {
-    return {
-      type: 'stripe',
-      paymentMethodId: customer.invoice_settings.default_payment_method,
-    };
-  } else if (subscriptions?.length) {
+  // First check if payment method is external_paypal
+  if (subscriptions?.length) {
```

Comment:
`firstListedSubscription` is not really used, and this nested `if` statement can be flattened to improve readability
```suggestion
  if (subscriptions?.length && subscriptions[0].collection_method === 'send_invoice') {
```